### PR TITLE
feat: SWF: identification metric nr. 5

### LINKED ISSUE #298: Identification summary workflow: implement annotation-dependent and other metrics
Ensure that annotation-dependent metrics (similar as the one in quantification summary workflow) and other metrics are implemented in identification summary workflow

parent issue: https://github.com/iRNA-COSI/APAeval/issues/239

Estimate: 4h

### PR DESCRIPTION
**Pull request by dominikburri**
Fixes #298 

Include metric nr. 5 (Percentage of genes with correctly identified number of PAS) and plot 3. "Bar plot visualizing percentage of genes with correctly identified number of PAS" in identification summary workflow.

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated corresponding READMEs (if applicable)
- [x] My code follows the templates/style guidelines of the repository
- [x] In- and output formats comply with APAeval specifications
- [x] No parameters or file names are hardcoded
- [x] Results, logs or other output is not commited to the repository
- [x] I have tested my code

## Notes

This branch is created from #432, which includes additional commits for the optional metrics.

### CODE REVIEW

**Review by ninsch3000 — CHANGES_REQUESTED** [review]
Workflow failed for me, I guess because pyranges is not available within a container (not sure). Please fix. Other than that it looks good to me, I didn't do any proper unit testing of the new functions though.

File: utils/apaeval/src/apaeval/main.py [hunk 1]
```diff
@@ -85,6 +90,7 @@ def find_pas_genes(gene, df):
     Returns:
         pd.Series: Array of indices indicating PAS for gene 'gene'.
     """
+    assert all([x in gene.keys() for x in ["seqname", "source", "start", "end"]])
```
[the remark this answers is not in the document]
Comment (dominikburri) [reply utils/apaeval/src/apaeval/main.py]: thanks, changed!

File: summary_workflows/identification/identification_dockers/i_metrics/requirements.txt [hunk 2]
```diff
@@ -3,4 +3,4 @@ jsonschema
 scipy
 sklearn
```
[the remark this answers is not in the document]
Comment (dominikburri) [reply summary_workflows/identification/identification_dockers/i_metrics/requirements.txt:4]: thanks, added `pyranges`. for me it worked before, maybe I included it in the conda package? I hope it works now.

**Review by ninsch3000 — APPROVED** [review]

**Review by faricazjj — APPROVED** [review]
LGTM thank you so much @dominikburri !!

**Pull request merged**